tests/integration/create.bats

```diff
@@ -39,3 +39,19 @@ function teardown() {
 
   testcontainer test_busybox running
 }
+
+@test "runc create --pid-file" {
+  runc create --pid-file pid.txt --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  testcontainer test_busybox created
+
+  # check pid.txt was generated
+  [ -e pid.txt ]
```

Comment:
Maybe we should also check that `pid` matches the pid shown in `runc list`? Or is that not really worth the effort?